### PR TITLE
Move package build results code out of responsive_ux

### PR DESCRIPTION
**Pull request by saraycp**
Since `responsive_ux` feature flag is completely rolled out, we are moving the related code out of the namespace.

In this case, for the package's build result area, I move the code and merge the new CSS rules with the existing ones.

~DO NOT MERGE until I squash the commits.~

-----------------

If this PR requires any particular action or consideration before deployment,
please check the reasons or add your own to the list:

* [ ] Requires a database migration that can cause downtime. Postpone deployment until maintenance window[1].
* [ ] Contains a data migration that can cause temporary inconsistency, so should be run at a specific point of time.
* [ ] Changes some configuration files (e.g. options.yml), so the changes have to be applied manually in the reference server.
* [ ] A new Feature Toggle[2] should be enabled in the reference server.
* [ ] Proper documentation or announcement has to be published upfront since the introduced changes can confuse the users.

[1] https://github.com/openSUSE/open-build-service/wiki/Deployment-of-build.opensuse.org#when-there-are-migrations
[2] https://github.com/openSUSE/open-build-service/wiki/Feature-Toggles-%28Flipper%29#you-want-real-people-to-test-your-feature

### CODE REVIEW

**Review by danidoni — APPROVED** [review]

**Comment by dmarcoux** [comment]
What is the reason for the `DO NOT MERGE` label? Waiting on #10861?

File: src/api/app/assets/stylesheets/webui/build-result.scss
Comment (dmarcoux) [inline src/api/app/assets/stylesheets/webui/build-result.scss:179]: [Newline missing at the end of file](https://thoughtbot.com/blog/no-newline-at-end-of-file)

File: src/api/app/assets/stylesheets/webui/application.scss
Comment (dmarcoux) [inline src/api/app/assets/stylesheets/webui/application.scss:36]: Just wondering, why is it singular now? I would keep this plural.

Comment (saraycp) [reply src/api/app/assets/stylesheets/webui/application.scss:36]: > What is the reason for the `DO NOT MERGE` label? Waiting on #10861?

The reason was explained in the description. I want to squash the commits before merging and I was waiting for at least one more review before doing so (to make it easier for the reviewer).

**Comment by saraycp** [comment]
> What is the reason for the `DO NOT MERGE` label? Waiting on #10861?

The reason was explained in the description. I want to squash the commits before merging and I was waiting for at least one more review before doing so (to make it easier for the reviewer).

**Comment by dmarcoux** [comment]
> > What is the reason for the `DO NOT MERGE` label? Waiting on #10861?
> 
> The reason was explained in the description.

Whoopsie, I totally overlooked that... sorry! :facepalm:

File: src/api/app/assets/stylesheets/webui/application.scss
Comment (saraycp) [reply src/api/app/assets/stylesheets/webui/application.scss:36]: The file under `responsive_ux` namespace was singular. I thought that maybe was a reason behind that change. But it's true that I would also go for a plural name. As we both agree, I'm going to rename it. :+1:

**Comment by saraycp** [comment]
@dmarcoux feedback addressed. As soon as you approve it I'll squash the commits

**Review by dmarcoux — APPROVED** [review]

**Comment by dmarcoux** [comment]
There are some typos in the commit message:  Remove non-responsive code of pac**ka**ge<strike>'s</strike> build results

Same for the commit description: The package<strike>'s</strike> build results **are** no longer under **the** responsive_ux feature
flag.

**Comment by saraycp** [comment]
> There are some typos in the commit message: Remove non-responsive code of pac**ka**ge's build results
> 
> Same for the commit description: The package's build results **are** no longer under **the** responsive_ux feature
> flag.

Corrected, thanks!

**Pull request merged**
